evaluate.py

```diff
@@ -16,47 +16,27 @@
 from tensorboardX import SummaryWriter
 from pytorch_monitor import monitor_module, init_experiment
 from loader import *
+from config import eval_config
+import json
 
 DEBUG_LOG = False
 
-CONFIG = dict(
-    # meta data
-    title="def2vec",
-    description="Translating definitions to word vectors",
-    run_name='ablate_attn', # defaults to START_TIME-HOST_NAME
-    run_comment='vanilla', # gets appended to run_name as RUN_NAME-RUN_COMMENT
-    log_dir='logs',
-    vocab_dim = 100,
-    vocab_source = '6B',
-    load_path = None,
-    # hyperparams
-    random_seed=42,
-    learning_rate=.0001,
-    max_epochs=15,
-    batch_size=16,
-    n_hidden=250,
-    # logging params
-    print_freq=1,
-    write_embed_freq=100,
-    eval_freq = 1000,
-    save_path="./checkpoints/full_run-vanillavanilla/epoch_5/model_weights.torch",
-    embedding_log_size = 10000,
-    # data loading params
-    num_workers = 8,
-    packing=False,
-    shuffle=True,
-    # model configuration [for ablation/hyperparam experiments]
-    weight_init="xavier",
-    input_method=INPUT_METHOD_ONE,
-    use_bidirection=True,
-    use_attention=True,
-    cell_type='GRU',
-    hidden_size=150,
-    embed_size=100,
-    dropout=0.1,
-    weight_decay=0.0,
-)
-TEST_FILE = 'data/glove/test_glove.%s.%sd.txt'%(CONFIG['vocab_source'],CONFIG['vocab_dim'])
+
+#load in the right config file from desired model to evaluate
+
+run_name = raw_input("run_name of model to eval: ")
```

Comment:
Instead of raw_input use argparse (quite standard in Python, see it in AI code all the time https://docs.python.org/2/library/argparse.html)